frontend/helpers/messageWrapper.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+
+import type {
+    MessageOptions,
+    MessageConfigOptions,
+    ConfigType,
+    ConfigDuration,
+    ConfigOnClose,
+} from 'ant-design-vue/types/message';
+import { message as antdMessage } from 'ant-design-vue';
+
+declare interface WrapperMessage {
+    success(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    warning(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    warn(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    info(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    error(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    loading(content: ConfigType, duration?: ConfigDuration, onClose?: ConfigOnClose): void;
+    open: (config: MessageOptions) => void;
+    config: (options: MessageConfigOptions) => void;
+    destroy: () => void;
+}
+
+export const message: WrapperMessage = {
```

Comment:
I no longer sure that was actually a good idea, probably adding `.catch(() => {})` to every `message` was enough. But let's keep it as it is, since it's already there

Reply:
Message actually doesn't have `catch`, only `then`. Not sure why

Reply:
> `MessageType` actually doesn't have `catch`, only `then`

Oh wow, sounds very strange 😱